pkgs/applications/misc/foxitreader/default.nix

```
@@ -0,0 +1,72 @@
+{ mkDerivation, lib, fetchzip, libarchive, patchelf, autoPatchelfHook, libsecret, libGL, zlib, openssl, qtbase, qtwebkit, qtxmlpatterns }:
+
+mkDerivation rec {
+  pname = "foxitreader";
+
+  version = "2.4.4.0911";
+  src = fetchzip {
+    url = "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/linux/${lib.versions.major version}.x/${
+        lib.versions.majorMinor version
+      }/en_us/FoxitReader.enu.setup.${version}.x64.run.tar.gz";
```

```suggestion
    url = "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/linux/${lib.versions.major version}.x/${lib.versions.majorMinor version}/en_us/FoxitReader.enu.setup.${version}.x64.run.tar.gz";
```

we usually just opt for the long URL because breaking it does not improve to much.